### PR TITLE
Cb 4 tools

### PR DESCRIPTION
**Pull request by ryannd**
* **Please check if the PR fulfills these requirements**
- [x] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)

Adds a tools component to show what tools we used to complete this project

* **What is the current behavior?What does the code do before you made the new changes?** 

This component shows the tools we used to complete this project. Before, the site did not show the tools used.

* **What is the new behavior?**

Allows the user to see what we used to make this project

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)

There is no breaking change introduced

* **Other information**:

### CODE REVIEW

File: frontend/src/components/Tools/Tools.js
Comment (Codubee) [inline frontend/src/components/Tools/Tools.js]: What do you think about using reactstrap [Container,Row,and Column](https://reactstrap.github.io/components/layout/) components instead of giving the classNames?

File: frontend/src/components/LandingPage/LandingPage.js
Comment (Codubee) [inline frontend/src/components/LandingPage/LandingPage.js]: When I run the app it says that it cant resolve this component. I think if you move the Tools component to the LandingPage folder and import the file it will work.

File: frontend/src/components/Tools/Tools.test.js
Comment (Codubee) [inline frontend/src/components/Tools/Tools.test.js]: Great job writing tests!

**Review by Codubee — CHANGES_REQUESTED** [review]
Great work Ryan! One change I ask is that since the tools component is related to the LandingPage, we move the Tools component to be inside of the LandingPage folder.

**Comment by Codubee** [comment]
Also there appears to be a merge conflict. Let me know if you need some help resolving it.

File: frontend/package.json
Comment (PretaxEnd9716) [inline frontend/package.json]: I think this is the reason why you had a merge conflict. Did you download another package? I think you used this command: `npm install --save enzyme enzyme-adapter-react-16 react-test-renderer jsdom`. Maybe you can revert the changes before you used the command and it will fix the merge error

**Review by PretaxEnd9716 — COMMENTED** [review]
The code seems good, minus the merge error.

File: frontend/src/components/LandingPage/LandingPage.js
Comment (kNgo8249) [reply frontend/src/components/LandingPage/LandingPage.js]: I believe to fix this problem you'll need to add another /Tools at the end so instead of looking at the Tools folder, the program will find the specific Tools.js file. Also I think you'll need to change the first part of the import to just one dot, so it looks for the Tools folder in the current directory (assuming you have moved the Tools folder into the LandingPage folder).

**Review by kNgo8249 — COMMENTED** [review]
Great job writing your code! As stated before be sure to move the Tools folder to the LandingPage folder.

File: frontend/src/components/Tools/Tools.js
Comment (ryannd) [reply frontend/src/components/Tools/Tools.js]: Sounds good, I haven't considered that. I will make those changes on my next commit.

File: frontend/package.json
Comment (ryannd) [reply frontend/package.json]: yeah, I don't think I ran npm update or something like that before pushing the branch. It's fixed now I think

File: frontend/src/components/LandingPage/LandingPage.js
Comment (ryannd) [reply frontend/src/components/LandingPage/LandingPage.js]: Yeah, I thought I updated the path after moving some things around but it looks like I forgot. Fixed now though.

File: frontend/src/index.js
Comment (Codubee) [inline frontend/src/index.js]: If you look a couple of lines below you will see that the min file has already been imported.

**Review by Codubee — CHANGES_REQUESTED** [review]
Great work Ryan! Just left one comment. I will approve once the merge conflicts have been resolved.

**Review by Codubee — APPROVED** [review]
LGTM!

**Pull request merged**
